### DISCUSSION TITLE
cannot import name 'FromOriginalControlnetMixin' from 'diffusers.loaders'

### DISCUSSION THREAD

**Discussion by dxmxyx** [huggingface/diffusers#9702] [Q&A] [answered]
which version of diffusers can solve it?

**Comment by asomoza** [comment, marked as answer]
Hi, if you want to use that [import](https://github.com/huggingface/diffusers/blob/1909952826ffb0609f5df172ac7948c46422614d/src/diffusers/loaders/__init__.py#L57), you need to use the [0.25.1](https://github.com/huggingface/diffusers/tree/v0.25.1) or older version.

**Reply by dxmxyx** [reply]
thank you! thats work!
